.github/workflows/continous-integration.yml

```diff
@@ -32,7 +42,8 @@ jobs:
         working-directory: examples
         run: ./gradlew clean build
       - name: Upload coverage to Codecov
-        # currently this doesn't work for forks
+        # currently this doesn't work for PR from forks
+        # see: https://github.com/codecov/codecov-action/issues/29
         if: matrix.java == 8
```

Comment:
Can we add a conditional on if this a fork somehow?

Reply:
I am unaware of any available check that would verify that

Reply:
I am unaware of any available check that would verify it.

Reply:
Would this be of any help, or is it at the wrong level of granualarity?
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
Oh, never mind. I misread the question to be about branches, not forks.

Reply:
What about checking `github.repository` or `github.base_ref`?

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context

Reply:
I did read that actions are disabled on forks by default, though. Fourth paragraph down in this section: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions#about-github-actions

Reply:
no luck, tried few combinations in https://github.com/ExpediaGroup/graphql-kotlin/pull/550

Following https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context, tried
- github.repository = ExpediaGroup/graphql-kotlin
- github.ref = `refs/pull/<PR_NUM>/merge`
- github.head_ref = fork branch name (without fork info)
- github.base_ref = master